src/css/base.scss

```diff
@@ -1,6 +1,15 @@
-/* This is an example of using Sass, otherwise plain CSS works too*/
-$primary-background: pink;
 
-body {
-  background-color: $primary-background;
+$primary-background: #4d4e4f;
+$sidebar-background: #9A9DAB;
+$card-background: #FFFFFF;
+$hydration-card: #5BC8AC;
+$activity-card: #DFE166;
+$sleep-card: #EC96A4;
+$border: 1px solid #000000;
+$primary-font: 'Open Sans', sans-serif;
+$secondary-font: sans-serif;
+
+@mixin flexed-element($direction) {
```

Comment:
Nice mixin for flex-box! Could add justify-content as an argument to this mixin.

Reply:
Mixin could also be used for media queries